nebula_ros/src/hesai/hesai_ros_wrapper.cpp

```diff
@@ -97,6 +96,12 @@ HesaiRosWrapper::HesaiRosWrapper(const rclcpp::NodeOptions & options)
       "Hardware connection disabled, listening for packets on " << packets_sub_->get_topic_name());
   }
 
+  auto hardware_id = hw_interface_wrapper_ && hw_interface_wrapper_->inventory()
```

Comment:
This is where it is potentially going to be more useful to use the device name (from the frame_id?)

Reply:
Changed to frame ID (see below). This has the added benefit of also working in offline replay :partying_face: 
![image](https://github.com/user-attachments/assets/87b56cee-4fd8-4ed7-9e99-15aa71be7b12)